packages/core/lib/resourceLoader.ts

```diff
@@ -43,3 +44,15 @@ export const describe: ResourceLoader = (iri, { env }) => {
 export const fromOwnGraph: ResourceLoader = (iri, { env }) => {
   return env.sparql.default.stream.query.construct(`CONSTRUCT { ?s ?p ?o } WHERE { GRAPH <${iri.value}> { ?s ?p ?o } }`)
 }
+
+const shorthandInserted = new WeakSet<Kopflos>()
+export async function insertShorthands(kopflos: KopflosInstance) {
```

Comment:
if I understand, there are no thread-safety issues because this is called only at start-up

Reply:
Yes, only done once